ccswbs/ugnext
# App Armor alerts

This PR adds App Armor alerts to ugnext similar to how it functions on doorknob.

Testing Plan:
- Let me know when you wish to start testing and I will create a test alert.
- Go to https://deploy-preview-19--ugnext.netlify.app/app-armor-test, and the alert dialog should appear.
- Click outside the modal to hide the alert
- Refresh the page, the alert should appear again
- Click the "Do not show me this again" button and refresh the page, the alert shouldn't appear again
- Close the window, and open the page in another window, the alert should appear again.
